044_PrimesInRange.js

```diff
@@ -0,0 +1,23 @@
+primesInRange = (start, end) => {
+  let num = start;
+  let primes = [];
+
+  primecheck = (x) => {
```

Comment:
Why are you not using x?

Reply:
I'm using it as a limiting condition of the loop and for checking whether the number is prime or not, this operation was enough.

Reply:
x used a limiting condition for the loop. In order to check the number is or not, using x again wasn't needed